src/Sylius/Bundle/CoreBundle/Tests/Responses/Expected/shipping_category/shipping_category.json

```diff
@@ -0,0 +1,8 @@
+{
+    "id": @integer@,
+    "name": "Regular",
+    "code": "reg",
```

Comment:
We should stick to one order of fields in a response. IMO `id`, `code`, `name`, etc. as below.